agent/async/lru_cache.go

```diff
@@ -24,6 +24,8 @@ type Cache interface {
 	Get(key string) (Value, bool)
 	// sets a value in cache. overrites any existing value
 	Set(key string, value Value)
+	// deletes the value from the cache
```

Comment:
lint: `Delete deletes ...` You can fix it for other methods as well while you're here.